README.md

```diff
@@ -5,15 +5,15 @@
 [![Coverage](https://sonarcloud.io/api/project_badges/measure?project=sonarscanner-msbuild&metric=coverage)](https://sonarcloud.io/component_measures?id=sonarscanner-msbuild&metric=coverage)
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](#contributing)
 
-The SonarScanner for .NET is the recommended way to launch a SonarQube or SonarCloud analysis for projects/solutions using MSBuild or dotnet command as build tool. It is the result of a [collaboration between SonarSource and Microsoft.](http://www.sonarqube.org/announcing-sonarqube-integration-with-msbuild-and-team-build/)
+The SonarScanner for .NET is the recommended way to launch a SonarQube or SonarCloud analysis for [Clean Code](https://www.sonarsource.com/solutions/clean-code/) projects/solutions using MSBuild or dotnet command as a build tool. It is the result of a [collaboration between SonarSource and Microsoft.]([http://www.sonarqube.org/announcing-sonarqube-integration-with-msbuild-and-team-build/](https://docs.sonarsource.com/sonarqube/latest/analyzing-source-code/scanners/sonarscanner-for-dotnet/))
```

Comment:
The modification of the 2nd link breaks the MD file and link is not displayed. I'm not sure what the intention was, as multiple different links seems to be mixed there

Reply:
The one that was there before didn't work (anymore)